fireapp/lib/domain/repository/reference_data_repository.dart

```diff
@@ -58,6 +58,32 @@ class ReferenceDataRepository {
     );
   }
 
+  Future<List<AssetType>> getAssetTypeHardCoded() async {
+    return Future.value([
+      AssetType(
+        id: 11,
```

Comment:
If you make these id's match the integers on line 78 of `scheduler_constraint_form_repository`, you won't need the mapping function